### PR TITLE
Removes SCP_294

### PR DESCRIPTION
**Pull request by ghost**
It's completely broken, hilariously easy to get, and you can print chems that the station should really not get

This isn't a full revert yet, But since MRTY was banned I can't see his changes to under so this might be awhile

### CODE REVIEW

**Comment by AlexTheSysop42** [comment]
>hilariously easy to get

It's nigh-impossible to find, if there's a trick people are using fix that instead

**Comment by Kraseo** [comment]
you can't even move it anymore thanks to someone making it not deconstructible

**Comment by Armhulen** [comment]
@AlexTheSysop42 It's a space ruin, bring a GPS.

@Kraseo why fix yet another one of it's unintended bugs for the nth time when we can just remove it.

**Comment by 81Denton** [comment]
>It's a space ruin, bring a GPS.

It doesn't have a GPS signal. If it even spawns you have to randomly bump into it

**Comment by Armhulen** [comment]
it's still hilariously strong giving changeling chems in infinite excess with no power restrictions

**Comment by Armhulen** [comment]
or just self antag/give out antag with gluttony blessing

**Comment by TheDreamweaver** [comment]
@Armhulen Glutton's blessing along with some other off limits chems can't even be produced from the machine. It has a blacklist.

**Comment by TheDreamweaver** [comment]
I normally agree with Arm on these sort of thing but it seems so more research needs to be done on the *actual* state of this. Given that it's location is unknown outside of pure luck, many of the game breaking chems are disabled, and now you can't even move it from it's location, I don't see this causing any issues that would make it remove worthy.

**Comment by Armhulen** [comment]
@TheDreamweaver Only the self antag toxin and the adminheal toxin are disabled. Not being able to move it is a bug.

**Comment by ShizCalev** [comment]
>hilariously easy to get into

are you talking about -after- the LASER TURRETS were fixed a couple days ago?

**Comment by 81Denton** [comment]
@Armhulen Blacklisted chems also affect the odysseus replicator, the whole list seems outdated in general if you look at some of the chems missing from it

**Comment by F-OS** [comment]
Just add chems to a blacklist/make it harder to get to, idk add mtf npcs and expand the ruin with more traps i guess.

**Comment by Xhuis** [comment]
scp-294 not scp_294 I bet you don't even read scp LOL

**Comment by Armhulen** [comment]
@xhuis DARK SOUL

**Comment by ShizCalev** [comment]
image

**Comment by optimumtact** [comment]
@ShizCalev I fully intend to merge this unless kor says otherwise

**Comment by optimumtact** [comment]
@KorPhaeron

**Comment by ishitbyabullet** [comment]
Why do people want this so badly?

It's only use to me is mutation toxin since xenobio was nerfed otherwise everyone powergames with it.

Some people say that we ruined it when we made it impossible to move it. You realize the shit you can get from this? It's powerful shit. The space ruin isn't even balanced enough since there is a way to easily fucking reach it still, I believe.

**Comment by ExcessiveUseOfCobblestone** [comment]
Fix it proper and have `closes #37326` if you don't want it merged...

**Comment by TheDreamweaver** [comment]
> Fix it proper and have closes #37326 if you don't want it merged...

Ah fuck, nvm just remove it

**Comment by ExcessiveUseOfCobblestone** [comment]
lol the idea of maintaining, the strongest deterrent.

**Comment by Armhulen** [comment]
if this is reopened i'll remove it from runtime

**Comment by TheDreamweaver** [comment]
I'll admit, I only ever used it for mutation toxins and healz, or for helping out other departments late game. While it was fun, I'll probably just move on to the next thing when this gets removed. 😞

**Comment by TheDreamweaver** [comment]
@Armhulen Alright, now what the fuck mate

**Comment by Armhulen** [comment]
@TheDreamweaver talk on discord k?

**Comment by Armhulen** [comment]
welp lets finish this

**Comment by TheDreamweaver** [comment]
I'd keep it in runtime, since it useful for code testing.  But ya, if you guys are so admament about this, then I won't argue further. Ave Maria my sweet machine friend...

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Remove the .ract file

**Comment by improvedname** [comment]
It doesnt have a gps

It just has 2 corpses with sec jumpsuit which in a recent steelpoint pr tm got changed to alwayz have their tracking sensors on so you just get a crew pinpointer and you get pointed to it

**Comment by AlexTheSysop42** [comment]
@improvedname that was fixed

**Comment by F-OS** [comment]
If that was fixed then just reduce the spawning chance for the love of god

**Comment by Bawhoppen** [comment]
Alternative to removing entirely, make it have limited uses (randomized) before it breaks down or runs out of charge. It's supposed to be inspired by SCP-294, rather than just copied from it, so it doesn't need to function exactly the same.

**Comment by TheDreamweaver** [comment]
@armhulen why though? Removing it from space ruins for balance reasons, sure. Removing it from runtime despite knowing coders use it for testing, what?

**Comment by Armhulen** [comment]
@optimumtact ready if this compiles

**Comment by TheDreamweaver** [comment]
Actually on second thought, I'd you're gonna remove it, replace it with an "admin dispenser" that allows you to dispense any chem in any amount. That'd would be even more useful to have for testing.

**Comment by Armhulen** [comment]
@optimumtact wait i'm wrong 1 more sec

**Comment by Armhulen** [comment]
ok now ready

File: code/modules/reagents/chemistry/machinery/scp_294.dm [hunk 1]
```diff
@@ -1,95 +0,0 @@
-//////////////////////////////////////////
-//				SCP 294					//
-//										//
-//	This is a child of a chemistry		//
-//	dispenser. Info of how it works at	//
-//	http://www.scp-wiki.net/scp-294		//
-//										//
-//////////////////////////////////////////
-
-/obj/machinery/chem_dispenser/scp_294
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/machinery/scp_294.dm]: >not leaving the machine itself in for shenanigans

Ishygddt

**Review by PKPenguin321 — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/machinery/scp_294.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/reagents/chemistry/machinery/scp_294.dm]: This is how you get catgirls.

Comment (KorPhaeron) [reply code/modules/reagents/chemistry/machinery/scp_294.dm]: add scp girls

**Pull request merged**